views/components/navbar.pug

```diff
@@ -9,7 +9,7 @@ mixin navbar
     .collapse.navbar-collapse(id='navbarSupportedContent')
       ul.navbar-nav.mr-auto
         li.nav-item
-          a.nav-link(href='#') All restaurants
+          a.nav-link(href='../restaurants') All restaurants
```

Comment:
Does this work? I think it should be `href='/restaurants'`

Reply:
Yeah this works on my com